geo/CHANGES.md

```diff
@@ -11,6 +11,7 @@
   * <https://github.com/georust/geo/pull/928>
 * Added `RemoveRepeatedPoints` trait allowing the removal of (consecutive)
   repeated points.
+* Added very basic `Offset` trait and implemented it for `Line` and `LineString`
```

Comment:
A minor nit: there's no need to qualify this as "very basic", since the docs will make the limitations clear.

Reply:
👍 I will try to reword it to be more specific, or be more clear about any limitations in the rest of the docs